packages/gatsby-plugin-page-creator/src/path-utils.ts

```diff
@@ -21,14 +21,19 @@ export function removeFileExtension(absolutePath: string): string {
   return absolutePath.replace(/\.[a-z]+$/, ``)
 }
 
-const slugPartsRegex = /(\{.*?\})/g
+// Remove trailing slash
+export function stripTrailingSlash(str: string): string {
+  return str.endsWith(`/`) ? str.slice(0, -1) : str
+}
+
+const curlyBracesContentsRegex = /(\{.*?\})/g
```

Comment:
While you're changing it anyways. You're not using the group so might as well drop it?

This regex has the same limits as above (`foo/{bar/{baz}/doo` -> `bar/{baz`)

```suggestion
const curlyBracesContentsRegex = /\{.*?\}/g
```